src/containers/AccountConnection.jsx

```diff
@@ -281,7 +281,7 @@ class AccountConnection extends Component {
                       {t('account.message.syncing.bill', { name: connector.name })}
                       <br />
                       <span className={styles['col-account-success-highlighted-data']}>
-                        {this.state.account.folderId}
+                        {this.state.account.auth.folderPath}
```

Comment:
The folder is (for now) set to a default one (waiting the folder path choice) and when you create a new account it will be set to the account from the state before reaching this code, so it will be here.
As this folderPath is stored in the base, when you fetch an account it will also be present. I don't see a case when it will be missing so it should work fine.